docs/reference/glossary-of-terms.md

```diff
@@ -49,6 +49,9 @@ DatatypeProperty relate terms to literal data (e.g., strings, numbers, datetimes
 #### Description Logic
 Description Logic (DL) are a family of formal knowledge representation languages. It provides a logical formalism for ontologies and is what [OWL](#OWL) is based on. DL querying can be used to query ontologies in Protege.
```

Comment:
I believe "Description Logic**s**" fits the definition provided. Description Logic (singular) would be one of the varieties from the "family" described (Attributive language, Existential language, etc). This is my understanding, but @henrietteharmse would have better insight.

Reply:
logics is write for the description :)